fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastore.java

```diff
@@ -231,6 +231,10 @@ public HivePartitionStats getTableStatistics(String dbName, String tblName) {
                 .map(FieldSchema::getName)
                 .collect(toImmutableList());
         List<ColumnStatisticsObj> statisticsObjs = client.getTableColumnStats(dbName, tblName, dataColumns);
+        if (statisticsObjs.isEmpty()) {
+            // Try to use spark unpartitioned table column stats
+            statisticsObjs = HiveMetastoreApiConverter.getColStatsFromSparkParams(table);
```

Comment:
I think it's best to catch and ignore any exceptions when processing optional column stats in table properties.

Reply:
Done. Thx.